models/public/retinaface-anti-covid/retinaface-anti-covid.md

```diff
@@ -0,0 +1,114 @@
+# RetinaFace-Anti-Covid
+
+## Use Case and High-Level Description
+RetinaFace-Anti-Cov is a customized one stage face detector to help people protect themselves from CovID-19. More details provided in the [paper](https://arxiv.org/abs/1905.00641) and [repository](https://github.com/deepinsight/insightface/tree/master/RetinaFaceAntiCov)
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
```

Comment:
Also you need to add types: detection, object attributes (or even facial landmarks).

Reply:
Done